src/resources/raids/policestation.ts

```diff
@@ -272,8 +272,8 @@ export const station: Location = {
 
 		{
 			type: 'LootChannel',
-			name: 'mall-fountain',
-			display: 'Mall Fountain',
+			name: 'Staff Room',
```

Comment:
names must follow discord channel naming rules (lowercase and use - instead of spaces)
```suggestion
			name: 'staff-room',
```